site/templates/film/item.jet

```diff
@@ -62,94 +80,106 @@
                     {{end}}
                   {{end}}
+                  {{if subtitles}}
+                    <div class="meta-detail-subtitle">
+                      <h2>{{i18n("meta_detail_subtitles")}}</h2>
+                      <p>{{subtitles}}</p>
+                    </div>
+                  {{end}}
+                  {{if captions}}
+                    <div class="meta-detail-subtitle">
+                      <h2>{{i18n("meta_detail_captions")}}</h2>
+                      <p>{{captions}}</p>
+                    </div>
+                  {{end}}
                 {{end}}
-                {{if subtitles}}
-                  <div class="meta-detail-subtitle">
-                    <h2>{{i18n("meta_detail_subtitles")}}</h2>
-                    <p>{{subtitles}}</p>
+                {{if len(film.Countries) > 0 }}
+                  <div class="meta-detail-country">
+                    <h2>{{i18n("meta_detail_countries", len(film.Countries))}}</h2>
+                    <p>{{ film.Countries }}</p>
                   </div>
                 {{end}}
-                {{if captions}}
-                  <div class="meta-detail-subtitle">
-                    <h2>{{i18n("meta_detail_captions")}}</h2>
-                    <p>{{captions}}</p>
+                {{if len(film.Studio) > 0 }}
+                  <div class="meta-detail-studio">
+                    <h2>{{i18n("meta_detail_studios", len(film.Studio))}}</h2>
+                    {*{ FORMAT STUDIOS }*}
+                    {{ studios := "" }}
+                    {{ range studio := film.Studio }}
+                      {{ studios = studios + studio + ", "}}
+                    {{ end }}
+                    {{ if len(studios) > 0 }}
+                      {{ studios = studios[0:len(studios) - 2] }}
+                    {{ end }}
+                    <p>{{ studios }}</p>
                   </div>
                 {{end}}
-              {{end}}
-              {{if len(film.Countries) > 0 }}
-                <div class="meta-detail-country">
-                  <h2>{{i18n("meta_detail_countries", len(film.Countries))}}</h2>
-                  <p>{{ film.Countries }}</p>
-                </div>
-              {{end}}
-              {{if len(film.Studio) > 0 }}
-                <div class="meta-detail-studio">
-                  <h2>{{i18n("meta_detail_studios", len(film.Studio))}}</h2>
-                  {*{ FORMAT STUDIOS }*}
-                  {{ studios := "" }}
-                  {{ range studio := film.Studio }}
-                    {{ studios = studios + studio + ", "}}
-                  {{ end }}
-                  {{ if len(studios) > 0 }}
-                    {{ studios = studios[0:len(studios) - 2] }}
-                  {{ end }}
-                  <p>{{ studios }}</p>
-                </div>
-              {{end}}
-            </div>
-          </s72-element-switcher>
+              </div>
+            </s72-element-switcher>
+            {{if isset(sponsorImage)}}
```

Comment:
If keen to keep the duplication, perhaps define it as a block and yield it twice.

Reply:
- Duplication strategy agreed by design and delivery until css grid refactor
- Changed to yield block